codespell_lib/data/dictionary.txt

```diff
@@ -5536,7 +5536,7 @@ mimatching->mismatching
 mimicing->mimicking
 mimick->mimic
 mimicks->mimics
-mimimise->minimise
+mimimise->minimize
```

Comment:
`minimise` is the British spelling of `minimize`

Reply:
I think there should be an entry for both.
```
mimimise->minimise
mimimize->minimize
```

Reply:
@Gelma 
Please add the suggestion above https://github.com/lucasdemarchi/codespell/pull/437/files#r181580550
and repush. Thanks.